src/utils/addresses.ts

```diff
@@ -0,0 +1,121 @@
+  '0x34ed0aa248f60f54dd32fbc9883d6137a491f4f3', // WonderzoneMeteorchaserCollection
+  '0xa8ee490e4c4da48cc1653502c1a77479d4d818de', // BinanceUsCollection
+  '0x09305998a531fade369ebe30adf868c96a34e813', // PMDreamverseEminence
+  '0x24d538a6265b006d4b53c45ba91af5ef60dca6cb', // CybermikeCyberSoldier
+  '0xe7a64f6a239ed7f5bf18caa1ce2920d0c1278129', // DCMeta
+  '0x5df4602e7f38a91ea7724fc167f0c67f61604b1e', // WZWonderbot
+  '0x7038e9d2c6f5f84469a84cf9bc5f4909bb6ac5e0', // DGFall2020
+  '0x30d3387ff3de2a21bef7032f82d00ff7739e403c', // MFSammichgamer
+  '0xb5d14052d1e2bce2a2d7459d0379256e632b855d', // SugarclubYumi
+  '0x54266bcf2ffa841af934f003d144957d5934f3ab', // EtheremonWearables
+  '0x60d8271c501501c4b8cd9ed5343ac59d1b79d993', // MLPekingopera
+  '0x90958d4531258ca11d18396d4174a007edbc2b42', // ChinaFlying
+  '0x480a0f4e360e8964e68858dd231c2922f1df45ef', // TechTribalMarc0matic
+  '0x5cf39e64392c615fd8086838883958752a11b486', // DigitalAlchemy
+  '0xc3ca6c364b854fd0a653a43f8344f8c22ddfdd26', // CZMercenaryMTZ
+  '0xb96697fa4a3361ba35b774a42c58daccaad1b8e1', // WonderzoneSteampunk
+  '0x102daabd1e9d294d4436ec4c521dce7b1f15499e', // DCNiftyblocksmith
+  '0xfeb52cbf71b9adac957c6f948a6cf9980ac8c907', // Halloween2020Collection
+  '0xecf073f91101ce5628669c487aee8f5822a101b1', // Xmas2020Collection
+  '0x1a57f6afc902d25792c53b8f19b7e17ef84222d5', // MemeDontBuyThis
+  '0xffc5043d9a00865d089d5eefa5b3d1625aec6763', // ReleaseTheKraken
+  '0xe1ecb4e5130f493551c7d6df96ad19e5b431a0a9', // 3LAUBasics
+  '0xdd9c7bc159dacb19c9f6b9d7e23948c87aa2397f', // XmashUp2020
+  '0x0b1c6c75d511fae05e7dc696f4cf14129a9c43c9', // MLLiondance
+  '0x4c290f486bae507719c562b6b524bdb71a2570c9', // AtariLaunch
+  '0x6b47e7066c7db71aa04a1d5872496fe05c4c331f', // RTFKTXAtari
+  '0x68e139552c4077ce5c9ab929c7e18ca721ffff00', // RACBasics
+  '0xc82a864a94db3550bc71fcb4ce07228bcec21f1a', // WinklevossCapital
+  '0x51e0b1afe5da0c038fc93a3fc8e11cf7a238b40b', // DGAtariDillonFranci
+]
+
+export const CURATORS_PAYMENT_ADDRESSES = [
+  '0x5d7846007c1dd6dca25d16ce2f71ec13bcdcf6f0',
+  '0x716954738e57686a08902d9dd586e813490fee23',
+  '0xc958f028d1b871ab2e32c2abda54f37191efe0c2',
+  '0x82d54417fc69681dc74a6c0c68c6dbad5a2857b9',
+  '0x9db59920d3776c2d8a3aa0cbd7b16d81fcab0a2b',
+  '0x91e222ed7598efbcfe7190481f2fd14897e168c8',
+  '0x6cdfdb9a4d99f16b5607cab1d00c792206db554e',
+  '0x862f109696d7121438642a78b3caa38f476db08b',
+  '0xc8ad6322821b51da766a4b2a82b39fb72b53d276',
+  '0xa8c7d5818a255a1856b31177e5c96e1d61c83991',
+  '0x336685bb3a96e13b77e909d7c52e8afcff1e859e',
+  '0x41eb5f82af60873b3c14fedb898a1712f5c35366',
+  '0x470c33abd57166940095d59bd8dd573cbae556c3',
+  '0x1dec5f50cb1467f505bb3ddfd408805114406b10',
+  '0x5ce9fb617333b8c5a8f7787710f7c07002cb3516',
+  '0x805797df0c0d7d70e14230b72e30171d730da55e'
+]
+
+export const GRANTS_REVENUE_ADDRESSES = [
+  '0xf22b4b9175f5b1a2730e84f5c2360a029a333fde', // https://governance.decentraland.org/proposal/?id=29b3a3a0-74fd-11ed-a9bf-f772a12a0556
```

Comment:
🧹  ?

Reply:
The comment? I think it's an useful way to quickly find out who owns that address